CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Upgraded `nx` from version `21.5.1` to `22.1.`
```

Comment:
* Use the exact `nx` version: `22.1.3`
* Move this entry to the end of the `### Changed` section